api/API-README.MD

```diff
@@ -250,8 +250,8 @@ All routes work with /api/recipient or /api/recipients
 | ------------------ | ------ | ------------------------------------------------- | ---------------------------------------- |
 | /api/recipients    | GET    | none                                              | none                                     |
 | /api/recipient/:id | GET    | none                                              | none                                     |
-| /api/recipient     | POST   | name, age, city, state, zip_code, veteran_status, | admin, program_manager, service_provider |
-|                    |        | household_size                                    |                                          |
+| /api/recipient     | POST   | first_name, last_name, age, household_id          | admin, program_manager, service_provider |
+|                    |        | veteran_status                                    |                                          |
 | /api/recipient/:id | PUT    | desired edit as key:value                         | admin, program_manager                   |
 | /api/recipient/:id | DELETE | none                                              | admin                                    |
 
```

Comment:
If you could, please change the returned object recipients examples to reflect the new changes, ie. name to first_name / last_name.